docs/source/plugin/bot.rst

```diff
@@ -220,32 +220,69 @@ Channels & users
 
 Knowing how to talk is good for a bot, but you may be wondering what the bot
 knows about the channels and their users. For that, you can use the bot's
-:attr:`~sopel.bot.Sopel.channels` attribute. For example, to list all channels
-the bot is in::
+:attr:`~sopel.bot.Sopel.channels` attribute.
+
+For example, to list all channels the bot is in::
 
     for name, channel in bot.channels.items():
         # do something with the name and the channel
 
+.. note::
+
+    Sopel doesn't know about channel it didn't join first, and it forgets
+    everything about channel it leaves.
+
+Getting a channel's information
+-------------------------------
+
+To get a channel's information, you need to know its name, with its channel
+prefix (usually ``#``), such as this::
+
+    channel = bot.channels['#channel_name']
+
 With the ``trigger`` object, you can also access the channel object directly
 (assuming the message comes from a channel, which you should check first)::
 
     channel = bot.channels[trigger.sender]
```

Comment:
"which you should check first" — so demonstrate how?

```suggestion
    if not trigger.sender.is_nick():
        channel = bot.channels[trigger.sender]
    else:
        # the `trigger` did not come from a channel
```

Reply:
I could have done that. Instead, I took the long road and actually document a bit more what it means to check if a message comes from a channel in my last commit. Hope you like it!